src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -57,14 +58,19 @@ function renderButton(sc, action, display, style, showText, asItem, autoFocus, t
       </Button>
     );
   } else {
+    if (display.ID === 'mute-audio' && display.state === 'disabled' && text === 'No microphone') {
+      ariaLabelText = 'No Microphone, disabled';
+    } else if (display.ID === 'mute-video' && display.state === 'disabled' && text === 'No camera') {
+      ariaLabelText = 'No Camera, disabled';
+    }
     output = (
       <Button
         className={sc('control-button')}
         type={controlTypeToButtonType[type] || 'default'}
         size={48}
         isDisabled={isDisabled}
         onClick={action}
-        ariaLabel={text}
+        ariaLabel={ariaLabelText || tooltip}
```

Comment:
Why do we need the or condition here with tooltip?

Reply:
Okay.  From next time please keep PRs separate. I beleive these PRs are independent so we should not mix the PR's codes.